<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>withdrawdll</class>
 <widget class="QDialog" name="withdrawdll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="TenEuro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>10 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TwentyEuro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>20 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FiftyEuro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>50 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HundredEuro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>100 €</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
